api/v1/models/MentorProjectIdea.js

```diff
@@ -0,0 +1,13 @@
+var Model = require('./Model');
+var MentorProjectIdea = Model.extend({
+	tableName: 'mentor_project_ideas',
+	idAttribute: 'id',
+	validations: {
```

Comment:
These are just ideas. I think the `name` field in #38 will be sufficient for those projects that get approved.